src/Console/ConfigurationResolver.php

```diff
@@ -253,7 +255,21 @@ public function getConfigFile()
     public function getDiffer()
     {
         if (null === $this->differ) {
-            $this->differ = false === $this->options['diff'] ? new NullDiffer() : new SebastianBergmannDiffer();
+            if (false !== $this->options['differ']) {
+                // 'diff' value does not matter here
```

Comment:
This _default_ means that `--diff` is equivalent to `--diff --differ=sbd`, not that `--differ=sbd` is equivalent to `--diff --differ=sbd`. One could expect that changing the diff type does not implicitly enables diff output.